src/Keypad.h

```diff
@@ -99,6 +99,13 @@ class Keypad : public Key {
 	bool keyStateChanged();
 	byte numKeys();
 
+// Testing methods
+	bool keysScanned() { return _scanned; }
+	bool keyFound() { return _found; }
+	bool stateChanged() { return _changed; }
+	byte countCols() { return _cols; }
```

Comment:
The name `countCols` starts with an imperative verb and sounds like you are telling it to count something. A name like `getColCount` would be a bit more descriptive and consistent with the typical get/set naming convention.